Fleet update command not enforcing minimum args.

anugup@Anurags-MBP ~ % calyptia update fleet --config-file dummy.conf 
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/calyptia/cli/cmd/fleet.NewCmdUpdateFleet.func1(0x1400015f800?, {0x14000275900?, 0x2?, 0x2?})
	/home/runner/work/cli/cli/cmd/fleet/update_fleet.go:35 +0x780
github.com/spf13/cobra.(*Command).execute(0x1400015f800, {0x140002758e0, 0x2, 0x2})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:940 +0x5c8
github.com/spf13/cobra.(*Command).ExecuteC(0x140000f8600)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x35c
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
main.main()
	/home/runner/work/cli/cli/main.go:16 +0x44

--- a/cmd/fleet/update_fleet.go
+++ b/cmd/fleet/update_fleet.go
@@ -23,9 +23,10 @@ func NewCmdUpdateFleet(config *cfg.Config) *cobra.Command {
 	completer := completer.Completer{Config: config}
 
 	cmd := &cobra.Command{
-		Use:   "fleet",
-		Short: "Update fleet",
-		Long:  "Update a fleet's shared configuration.",
+		Use:               "fleet",
+		Short:             "Update fleet by name",
+		Long:              "Update a fleet's shared configuration.",
+		Args:              cobra.ExactArgs(1),
 		ValidArgsFunction: completer.CompleteFleets,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			var err error

